tests/postgres_tests/test_aggregates.py

```diff
@@ -146,6 +146,16 @@ def test_string_agg_distinct_true(self):
         self.assertEqual(values['stringagg'].count('Bar'), 1)
 
 
+class TestArrayAggregateDistinct(TestStringAggregateDistinct):
```

Comment:
Oh right, sorry, I just understood your previous comment on this file! I will fix them now. I'm using the github editor to do this, and there is no way to squash commits (apart from when merging). I can do it locally when I have access to my machine later if it's an issue.